src/modules/85-iacm/components/IACMPipelineSteps/IACMApproval/__tests__/StepMode.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createRef } from 'react'
+import { render, act, screen } from '@testing-library/react'
+
+import { StepViewType, StepFormikRef } from '@pipeline/components/AbstractSteps/Step'
+import { TestWrapper } from '@common/utils/testUtils'
+import StepMode from '../StepMode'
+
+describe('IACMApprovalStep', () => {
+  test('renders correctly', () => {
+    const { getByText } = screen
+    render(
+      <TestWrapper>
+        <StepMode
+          stepViewType={StepViewType.Edit}
+          initialValues={{ timeout: '10s', name: '', identifier: '' }}
+          allowableTypes={[]}
+        />
+      </TestWrapper>
+    )
+
+    expect(getByText('Name')).toBeInTheDocument()
+    expect(getByText('pipelineSteps.timeoutLabel')).toBeInTheDocument()
```

Comment:
Sorry, I probably should have been a bit more specific. The suggestion (by the author of Testing Library) is to use `screen.getByX`, not to extract it.
```suggestion
    render(
      <TestWrapper>
        <StepMode
          stepViewType={StepViewType.Edit}
          initialValues={{ timeout: '10s', name: '', identifier: '' }}
          allowableTypes={[]}
        />
      </TestWrapper>
    )

    expect(screen.getByText('Name')).toBeInTheDocument()
    expect(screen.getByText('pipelineSteps.timeoutLabel')).toBeInTheDocument()
```

Not a blocker whichever way you want to do it.